lighthouse-core/audits/time-to-interactive.js

```diff
@@ -57,25 +57,28 @@ class TTIMetric extends Audit {
    * @return {!Promise<!AuditResult>} The score from the audit, ranging from 0-100.
    */
   static audit(artifacts) {
-    const trace = artifacts.traces[Audit.DEFAULT_PASS];
-    const pendingSpeedline = artifacts.requestSpeedline(trace);
-    const pendingFMP = FMPMetric.audit(artifacts);
+    return new Promise((resolve, reject) => {
+      const trace = artifacts.traces[Audit.DEFAULT_PASS];
+      const pendingSpeedline = artifacts.requestSpeedline(trace);
+      const pendingTracingModel = artifacts.requestTracingModel(trace);
+      const pendingFMP = FMPMetric.audit(artifacts);
+
+      return resolve(TTIMetric.calculate(pendingSpeedline, pendingTracingModel, pendingFMP, trace));
```

Comment:
This handoff is a little awkward as I wanted to preserve the indentation for the moment.
Once it's reviewed, I'll move the `Promise.all` up here and slide the body of `calculate` over a tab.